### PR TITLE
Update License and increment build

### PR DESCRIPTION
**Pull request by lewismc**
Checklist
* [X] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by lewismc** [comment]
@conda-forge-admin, please rerender

**Comment by lewismc** [comment]
@tboggs FYI... I needed to increment the version number :)
I hope this fixes the license issue it creates a new unique build.

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Summary: A Python module for hyperspectral image processing.
 
 A pure Python module for processing hyperspectral image data (imaging spectroscopy data).
 It has functions for reading, displaying, manipulating, and classifying hyperspectral
-imagery. SPy is Free, Open Source Software (FOSS) distributed under the MIT License..
+imagery. SPy is Free, Open Source Software (FOSS) distributed under the GNU General Public License.
```
Comment (lewismc) [inline README.md]: @tboggs as you can see.... again the bot has picked up the incorrect license from somewhere.

Comment (lewismc) [reply README.md]: @conda-forge-admin can you please take a look and advise on why this is happening/ Thank you

Comment (tboggs) [reply README.md]: @lewismc Yes, I see it. This page indicates that it is waiting for my review to accept the PR. Should I merge it (again) or wait for @conda-forge-admin to chime in here?

Comment (lewismc) [reply README.md]: Let's wait for @conda-forge-admin to chime in here?

Comment (tboggs) [reply README.md]: @conda-forge-admin can you please provide some guidance here? We're at a loss as to what keeps overwriting the README.

**Pull request merged**